src/Discord/Helpers/Bitwise.php

```diff
@@ -160,4 +162,17 @@ public static function floatCast($value)
 
         return $result;
     }
+
+    public static function is32BitWithGMP(): bool
```

Comment:
I'm not crazy about this function being called every single time something would need to use GMP just to determine if the extension is active or not. This should be done only once (preferably on discord __construct()) and the result stored for checking later. This function currently contains branching logic and is considered slow.